_pages/clients/more-speech.md

```diff
@@ -0,0 +1,28 @@
+---
+title: more-speech
+permalink: /more-speech/
+web: 
+github: unclebob/more-speech
+instances:
+tags:
+- social
+- client
+- desktop 
```

Comment:
We could add a different parameter for platform. Adding this dimension is a good idea to later filter by it.

```suggestion
platforms:
- desktop 
```